apps/frontend/src/app/5_pages/AavePage/components/BorrowAssetsList/components/BorrowModal/BorrowForm.tsx

```diff
@@ -52,19 +52,20 @@ export const BorrowForm: FC<BorrowFormProps> = ({ asset }) => {
     return Decimal.from(reserve.variableBorrowAPR).mul(100);
   }, [reserve]);
 
+  const borrowUsdAmount = useMemo(() => {
+    return borrowSize.mul(reserve?.priceInUSD ?? 0);
+  }, [borrowSize, reserve]);
```

Comment:
use depth deps in reserve in order to detect changes 
[borrowSize, reserve?.priceInUSD]);

Reply:
In this case reserve will change as a whole every time so no problem. But! That knowledge comes from looking inside the working of the hook, and better not make assumptions. So I'll totally change it. Thanks!